### PR TITLE
[DCA-8] AWS account for Schematic app

### PR DESCRIPTION
**Pull request by zaro0508**
Create an AWS account to run Schematica appp[1]

[1] https://github.com/Sage-Bionetworks/schematic

### CODE REVIEW

**Review by xschildw — APPROVED** [review]
As long as TBD is valid value for costCenter...

File: org-formation/organization.yaml [hunk 1]
```diff
@@ -567,6 +568,21 @@ Organization:
...
+    Properties:
+      AccountName: org-sagebase-schematic-prod
+      RootEmail: aws-schematic-prod@sagebase.org
+      Alias: org-sagebase-schematic-prod
+      Tags:
+        <<: !Include ./_default_org_tags.yaml
+        Department: FAIR
+        Project: DCA
+        AccountOwner: milen.nikolov@sagebase.org
+        CostCenter: "TBD"
```
Comment (zaro0508) [inline org-formation/organization.yaml:582]: @brucehoff @milen-sage please check Department/Project/AccountOwner and CostCenter are correct.

**Review by brucehoff — CHANGES_REQUESTED** [review]
I though we agreed just to use the 'dca-prod' account for both the DCA app' and the Schematic APIs.  The account was already set up, under DCA-8.  What is the benefit of a separate AWS account fir one part of the system?

**Comment by zaro0508** [comment]
@brucehoff we had discussions but i don't remember us making a decision one way or the other.  The benefits of separate account for schematic is to provide full security isolation of the two apps.  This could be important if we think that (one day) the apps would have a separate set of developers managing and maintaining them.  If that's not important then deploying both to the same AWS account is fine.

**Comment by brucehoff** [comment]
> @brucehoff we had discussions but i don't remember us making a decision one way or the other. The benefits of separate account for schematic is to provide full security isolation of the two apps. This could be important if we think that (one day) the apps would have a separate set of developers managing and maintaining them. If that's not important then deploying both to the same AWS account is fine.

i think the two app's will always be maintained by the same team and so it's OK to deploy them together.  We can check with @milen-sage.

**Comment by zaro0508** [comment]
OK.  no need to create new aws account so closing this PR.

**Pull request closed without merging**